app/helpers/claims_helper.rb

```diff
@@ -73,4 +73,12 @@ def subject_list(subjects)
       two_words_connector: connector
     )
   end
+
+  def claim_school_title(searching_for_additional_school)
```

Comment:
Feels like this method could have a more appropriate name than "claim school title". Is it more a question that happens to be used in the title?